pkg/events/derive/derive.go

```diff
@@ -42,20 +42,23 @@ func (t Table) Register(deriveFrom, deriveTo events.ID, deriveCondition func() b
 
 // DeriveEvent takes a trace.Event and checks if it can derive additional events from it as defined by a derivationTable.
 func (t Table) DeriveEvent(event trace.Event) ([]trace.Event, []error) {
-	derivatives := []trace.Event{}
-	errors := []error{}
+	derivedEvents := []trace.Event{}
+	deriveErrors := []error{}
 	deriveFns := t[events.ID(event.EventID)]
 	for id, deriveFn := range deriveFns {
 		if deriveFn.Enabled() {
-			derivative, errs := deriveFn.DeriveFunction(event)
+			derivs, errs := deriveFn.DeriveFunction(event)
```

Comment:
I think `derivatives` is a better name, no?

Reply:
Yeah, we can revert those names.